modules/EvseManager/IECStateMachine.cpp

```diff
@@ -372,6 +375,7 @@ void IECStateMachine::set_pwm_F() {
 
 // The higher level state machine in Charger.cpp calls this to indicate it allows contactors to be switched on
 void IECStateMachine::allow_power_on(bool value, types::evse_board_support::Reason reason) {
+    EVLOG_info << "Allow power on: " << value << " Reason: " << reason;
```

Comment:
Is that log message desired here or left over from debugging?

Reply:
It is desired